docs/src/modules/components/AppFrame.js

```diff
@@ -146,6 +167,35 @@ class AppFrame extends React.Component {
                   )}
                   <div className={classes.grow} />
                   <AppSearch />
+                  <Tooltip title="Change docs language" enterDelay={300}>
+                    <IconButton
+                      color="inherit"
+                      aria-owns={languageMenu ? 'language-menu' : undefined}
+                      aria-haspopup="true"
+                      onClick={this.handleLanguageIconClick}
+                    >
+                      <LanguageIcon />
+                    </IconButton>
+                  </Tooltip>
+                  <Menu
+                    id="language-menu"
+                    anchorEl={languageMenu}
+                    open={Boolean(languageMenu)}
+                    onClose={this.handleLanguageMenuClose}
+                  >
+                    <MenuItem
+                      selected={userLanguage === 'en'}
+                      onClick={() => this.handleLanguageMenuItemClick('en')}
+                    >
+                      English
+                    </MenuItem>
+                    <MenuItem
+                      selected={userLanguage === 'zh'}
+                      onClick={() => this.handleLanguageMenuItemClick('zh')}
+                    >
+                      Chinese
+                    </MenuItem>
+                  </Menu>
                   <Tooltip title="Edit docs colors" enterDelay={300}>
```

Comment:
We gonna have a space issue on mobile. I think that we can use the Hidden component to hide this one.

Reply:
Perhaps better to hide the github one? (It already feels a bit out of place, as the others are docs controls.

Reply:
Actually, it seems fine on mobile. I still wouldn't mind moving the github icon at some point.

Reply:
In this case, let's add some G.A event, see what's the less used button and hide it.

Reply:
Okay, I'll submit a separate PR for that.